shared/resources/Settings.bundle/en.lproj/Root.strings

```diff
@@ -1,44 +0,0 @@
-// SECTION: Root //
-"GROUP_TITLE_ACCOUNT"="Account";
-/* group label for Account settings */
-"LOGOUT_SELECTOR_TITLE"="Logout Now";
-/* allows user to log out of account*/
-"GROUP_TITLE_DEBUG"="Debug";
-/* a section title for debug settings (SEND_LOGS_SELECTOR_TITLE) */
-"SEND_LOGS_SELECTOR_TITLE"="Send Logs";
-/* if true, app will send debug logs to Salesforce Support */
-"GROUP_TITLE_CONNECTION"="Connection";
-/* Settings label for network connection */
```

Comment:
Before I removed this, I did a search for all these strings and could not find it anywhere in the workspace.